heliopy/util/config.py

```diff
@@ -4,6 +4,7 @@
 import configparser
 import os
 import heliopy
+from pathlib import Path
```

Comment:
Can this be `import pathlib`, and then use `pathlib.Path` below? It keeps the imports simpler.